newbank/server/NewBank.java

```diff
@@ -1,14 +1,16 @@
 package newbank.server;
 
 import java.util.HashMap;
-
+import static java.lang.System.out;
 public class NewBank {
 	
 	private static final NewBank bank = new NewBank();
 	private HashMap<String,Customer> customers;
-	
+	private HashMap<String,Integer> account;
```

Comment:
Might be better to change Integer for Double

Reply:
changes have been applied